code/modules/hallucination/fake_chat.dm

```diff
@@ -29,6 +29,9 @@
 	if(is_radio)
 		var/list/humans = list()
 		for(var/mob/living/carbon/human/existing_human in GLOB.alive_mob_list)
+			// these roles usually have special names that can reveal antags
+			if(IS_WIZARD(existing_human) || IS_NUKE_OP(existing_human) || IS_SPACE_NINJA(existing_human))
```

Comment:
Or abductors or nightmares or ashwalkers or golems or comms agents etcetera

Could you make it loop through the crew manifest instead? This is gonna keep breaking when new stuff keeps getting added (I've already seen this exact PR a few times)

Reply:
Accomplished with https://github.com/tgstation/tgstation/pull/72539/commits/3b6cdb89e2c6d9d9aacc0b9f4fde347f310ea4b4